typescript/examples/event_processor/config.yaml.example

```diff
@@ -0,0 +1,6 @@
+indexer_name: "event_processor"
```

Comment:
Ooop yeah my bad, I meant to remove this since I removed it from the config, the processors now report their own name.

As for multi processor deployment, yeah probably, though I wonder if putting them all in the same binary makes sense. I think having different build targets for each and they use shared code would maybe be better rather than having all part of one binary. Each has trade offs, we can discuss.

Reply:
Yeah probably simplest to have them all in one binary and worry about perf / size issues down the line. We could avoid the binary nightmare to some extent by having them all in the same image but let's not worry about any of that for now.